src/domain/accounts-ips/ip-metadata-validator.ts

```diff
@@ -6,21 +6,20 @@ import {
 } from "@domain/errors"
 
 export const IPMetadataValidator = ({
-  denyIPCountries,
-  allowIPCountries,
+  denyCountries,
+  allowCountries,
   denyASNs,
   allowASNs,
 }: IPMetadataValidatorArgs): IPMetadataValidator => {
-  const validateForReward = (ipMetadata?: IPType): true | ValidationError => {
+  const validate = (ipMetadata?: IPType): true | ValidationError => {
```

Comment:
There's this comment still about whether `validate` is the right name here https://github.com/GaloyMoney/galoy/pull/2905#discussion_r1267839895

Reply:
IMO this is an `authorisation` type concern. Not a `validation` we could have a `valid` IP that we don't want to  `allow`.

Reply:
so what is the suggestion (code change needed)? because the logic to validate (`validate` function) is the same but the config used `IPMetadataValidatorArgs` is different